src/api/trajserv/TrajservAPI.test.js

```diff
@@ -0,0 +1,220 @@
0,"n":"Ilanz","wa":0},{"p":[9.233073136,46.77847982],"at":268200000,"dt":268200000,"ap":1593581220,"dp":1593581220,"ad":60000,"dd":60000,"c":0,"sid":"8509170","dot":3,"put":3,"n":"Castrisch","wa":0},{"p":[9.275473618,46.79176617],"at":271200000,"dt":271200000,"ap":1593581520,"dp":1593581520,"ad":60000,"dd":60000,"c":0,"sid":"8509169","dot":3,"put":3,"n":"Valendas-Sagogn","wa":0},{"p":[9.310292318,46.8074042],"at":274200000,"dt":274800000,"ap":1593581820,"dp":1593581880,"ad":60000,"dd":0,"c":0,"sid":"8509168","dot":0,"put":0,"n":"Versam-Safien","wa":0},{"p":[9.35946722,46.81647561],"at":277800000,"dt":277800000,"ap":1593582180,"dp":1593582180,"ad":60000,"dd":60000,"c":0,"sid":"8509167","dot":3,"put":3,"n":"Trin","wa":0},{"p":[9.412610429,46.82359703],"at":281400000,"dt":281400000,"ap":1593582540,"dp":1593582540,"ad":60000,"dd":60000,"c":0,"sid":"8509183:3","dot":0,"put":0,"n":"Reichenau-Tamins","wa":0},{"p":[9.430388089,46.82862479],"at":282600000,"dt":282600000,"ap":1593582660,"dp":1593582660,"ad":60000,"dd":60000,"c":0,"sid":"8509182:2","dot":3,"put":3,"n":"Ems Werk","wa":0},{"p":[9.453115465,46.83316658],"at":284400000,"dt":284400000,"ap":1593582840,"dp":1593582840,"ad":0,"dd":0,"c":0,"sid":"8509181:1","dot":0,"put":0,"n":"Domat/Ems","wa":0},{"p":[9.512682752,46.84998445],"at":287400000,"dt":287400000,"ap":1593583140,"dp":1593583140,"ad":0,"dd":0,"c":0,"sid":"8509050:2","dot":3,"put":3,"n":"Chur West","wa":0},{"p":[9.528403269,46.85353538],"at":289800000,"dt":300600000,"ap":1593583380,"dp":1593584460,"ad":0,"dd":0,"c":0,"sid":"8509000:12","dot":0,"put":0,"n":"Chur","wa":0},{"p":[9.531529406,46.86189572],"at":301200000,"dt":301800000,"ap":1593584520,"dp":1593584580,"ad":0,"dd":0,"c":0,"sid":"8509006","dot":3,"put":3,"n":"Chur Wiesental","wa":0},{"p":[9.53319129,46.87697298],"at":303000000,"dt":303000000,"ap":1593584700,"dp":1593584700,"ad":0,"dd":0,"c":0,"sid":"8509051","dot":3,"put":3,"n":"Haldenstein","wa":0},{"p":[9.559202009,46.91877102],"at":306000000,"dt":306600000,"ap":1593585000,"dp":1593585060,"ad":0,"dd":0,"c":0,"sid":"8509053:1","dot":0,"put":0,"n":"Untervaz-Trimmis","wa":0},{"p":[9.559502944,46.93483522],"at":307800000,"dt":307800000,"ap":1593585180,"dp":1593585180,"ad":0,"dd":0,"c":0,"sid":"8509054","dot":3,"put":3,"n":"Zizers","wa":0},{"p":[9.564686224,46.94983066],"at":309000000,"dt":309000000,"ap":1593585300,"dp":1593585300,"ad":0,"dd":0,"c":0,"sid":"8509055","dot":3,"put":3,"n":"Igis","wa":0},{"p":[9.560949232,46.95724403],"at":310200000,"dt":310200000,"ap":1593585420,"dp":1593585420,"ad":0,"dd":0,"c":0,"sid":"8509056","dot":3,"put":3,"n":"Landquart Ried","wa":0},{"p":[9.554481362,46.96747635],"at":312600000,"dt":312600000,"ap":1593585660,"dp":1593585660,"ad":0,"dd":0,"c":0,"sid":"8509002:6","dot":3,"put":3,"n":"Landquart","wa":0}],"a":{"n":"RhB (Rhätische Bahn)","u":"http://www.sbb.ch","l":"","p":"","f":"","tz":"Europe/Berlin"},"f":{"n":"SBB","u":"http://www.sbb.ch","l":"de","s":"20191215","e":"20201212","v":"Fahrplan 2020"}}',
```

Comment:
same here

Reply:
Moved to a file. Continue in discussion https://github.com/geops/mobility-toolbox-js/pull/12#discussion_r448182290